app/src/main/java/org/thunderdog/challegram/component/user/UserView.java

```diff
@@ -305,6 +320,21 @@ protected void onDraw (Canvas c) {
       }
       avatarReceiver.draw(c);
     }
+    /*FIXME: never call tdlib.chat(...) directly in onDraw
+    TdApi.Chat rawChat = tdlib.chat(user.getChatId());
```

Comment:
This method is too heavy for the `onDraw` method. The state of `view` should be remembered at the moment `user` value is set.